playdate_example/src/playdate_example.nim

```diff
@@ -93,8 +93,8 @@ proc handler(event: PDSystemEvent, keycode: uint) {.raises: [].} =
         font = try: playdate.graphics.newFont(FONT_PATH) except: nil
```

Comment:
Altough we are not changing the API here, I'd still recommend targeting the `dev` branch for this PR so that we can track any new bugs to a specific release and have a sense of what is in the public release and what isn't

Reply:
<img width="489" alt="image" src="https://github.com/user-attachments/assets/af890825-6036-4fac-80aa-522cbb7f23cd">
You can change it by pressing the edit button next to the PR title and then using the dropdown